src/core/server/saved_objects/service/lib/utils.ts

```diff
@@ -80,4 +80,11 @@ export class SavedObjectsUtils {
     total: 0,
     saved_objects: [],
   });
+
+  public static filterWorkspacesAccordingToBaseWorkspaces(
+    targetWorkspaces?: string[],
+    baseWorkspaces?: string[]
+  ): string[] {
+    return targetWorkspaces?.filter((item) => !baseWorkspaces?.includes(item)) || [];
```

Comment:
Would you mind change to use `xor` like `_.xor([(targetWorkspaces || [].concat(baseWorkspaces || [])), baseWorkspaces || []])`?

Reply:
We may not because OSD is deprecating `Lodash`.